src/js/vendor/highlight.bundle.js

```diff
@@ -26,6 +26,7 @@
   hljs.registerLanguage('scala', require('highlight.js/lib/languages/scala'))
   hljs.registerLanguage('shell', require('highlight.js/lib/languages/shell'))
   hljs.registerLanguage('sql', require('highlight.js/lib/languages/sql'))
```

Comment:
Here's my new suggestion. Add `sql` as an alias for `sqlpp`, as discussed in the comments.

```suggestion
  hljs.registerLanguage('sql', require('highlight.js/lib/languages/sqlpp'))
```

Reply:
As you said, we can probably live with the merging sql and sql++ under the same highlighter (not ideal, but there are very close construct-wise), but I think I would prefer to go with sqlpp as this is something that developers are used to since you don't tend to use symbols in filename extensions.

c++ ==> cpp
c# ==> cs

Not all highlighters use the filename extension, but none of them use symbols in the highlighter name. 

I imagine it becomes a bit trickier in our case because sql++ is pretty new, so there hasn't really been much of a call for worrying about what to do about the file extension.

Though AsterixDB seems to have gone with convention and plumped for sqlpp --> https://asterix-gerrit.ics.uci.edu/plugins/gitiles/asterixdb/+/refs/changes/05/10705/1/asterixdb/asterix-app/src/test/resources/runtimets/queries_sqlpp/tpcds/q77

Reply:
Ah sorry, I may not have explained myself clearly 😃 I'm not talking about the file extension for SQL++ files, but the text label that appears above code listings in the rendered HTML documents. I want this label to say `SQL++`, to minimize the potential for new user confusion. If there's a non-hacky way of getting the text label to say `SQL++` then I'd happily go for that &mdash; perhaps with an Antora extension?

We agreed in the planning for N1QL --> SQL++ that we would not change the string `n1ql` in file names or directory names: this also applies to example files.

Reply:
Further to this @RayOffiah &mdash; what does [this line in 07-copy-to-clipboard.js](https://github.com/couchbase/docs-ui/blob/master/src/js/07-copy-to-clipboard.js#L3) do? Would this be useful?

Reply:
I'm not too familiar with the internals of highlight.js, but I don't think so. The problem seems to be registering the name `sql++`, and this line seems related to setting parameters for removing highlight code when copying the source text (that's a very much a guess). 

Though again, they seem to have registered text only, and not used symbols such as `++` to try and reference them.

Reply:
G'morning.

@simon-dew 

I'm not too familiar with the internals of highlight.js, but I don't think so. The problem seems to be registering the name `sql++`, and this line seems related to setting parameters for removing highlight code when copying the source text (that's a very much a guess). 

Though again, they seem to have registered text only, and not used symbols such as `++` to try and reference them.